components/gallery/useGalleryItem.ts

```diff
@@ -118,7 +118,7 @@ export const useGalleryItem = (nftId?: string): GalleryItem => {
     nftResources.value = resources
 
     const asset = whichAsset(nftMetadata.value)
-    nftImage.value = asset.image
+    nftImage.value = `${asset.image}?original=true`
```

Comment:
I think this will increase loading time on some collection (i.e. donkey on rmrk) where 1 img is +5mb

Reply:
This is only used in gallery items, right?

Reply:
double check, yes